org.jacoco.doc/docroot/doc/faq.html

```diff
@@ -141,7 +141,7 @@
   <code>-Xshareclasses:none</code>.
 </p>
 
-<h3>Why do I get an error "<code>ClassNotFoundException: org.jacoco.agent[...]Offline</code>"?</h3>
+<h3>Why do I get an error "<code>NoClassDefFoundError: org.jacoco.agent[...]Offline</code>"?</h3>
```

Comment:
@marchof I was thinking about addition rather than replacement - like you say in commit message

> should also mention

And I'm really interested in your comments regarding following paragraph from description of ticket:

> maybe we can wrap access to class into try-catch to provide more user-friendly error message

?